zuul.d/architecture-jobs.yaml

```diff
@@ -9,19 +9,25 @@
     name: cifmw-architecture-validate-base
     parent: cifmw-base-minimal
     vars:
-      cifmw_networking_mapper_networking_env_def_path: >-
-        {{
-          [ansible_user_dir,
-           zuul.projects['github.com/openstack-k8s-operators/ci-framework'].src_dir,
-           'ci/playbooks/files/networking-env-definition.yml']
-           | path_join
-        }}
       cifmw_architecture_repo: >-
         {{
           [ansible_user_dir,
            zuul.projects['github.com/openstack-k8s-operators/architecture'].src_dir]
            | path_join
         }}
+      cifmw_networking_env_def_file_local: >-
+        {{
+          omit if not cifmw_networking_env_def_file else
```

Comment:
I'm pretty sure that `omit` here will end in the variable having the omit placeholder string instead of the variable not defined.